WgWall/ClientApp/src/app/products/products.component.ts

```diff
@@ -23,11 +23,11 @@ export class ProductsComponent {
 
   //product lists
   private products: Product[]
-  public register: String[];
+  public register: string[];
   public active: Product[];
 
   //input
-  public newProductName: String = ""
+  public newProductName: string = ""
   public isLoading: Boolean = false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this 'Boolean' wrapper object with primitive type 'boolean'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1533)